javascript/javascript1/week1/script.js

```diff
@@ -0,0 +1,68 @@
+//Age calculator
+
+let yearOfBirth = 1995
+let yearFuture = 2027
+let age = yearFuture - yearOfBirth
+console.log("You will be " + age + " years old in " + yearFuture)
+
+//Goodboy-oldboy
+
+let dogYearOfbirth = 2018;
+let dogYearFuture = 2027;
+let dogYear = yearFuture - yearOfBirth;
+let shouldShowResultInDogYears = true;
+
+if (shouldShowResultInDogYears) {
+    let dogInDogYears = dogYear * 7;
+    console.log("Your dog will be " + dogInDogYears + " dog years old in 2027");
+} else {
+    console.log("Your dog will be" + dogYear + "human years old in 2027");
+}
```

Comment:
Same, use `const` instead of `let`, as you don't change these variables further in code.